test-end-to-end/webapp-cypress/cypress/e2e/004_012_peer_marking_for_individual_assignment.cy.js

```diff
@@ -0,0 +1,124 @@
+import setDate from '../support/setDate'; //https://github.com/cypress-io/cypress/issues/1366#issuecomment-437878862
+describe('004_012_peer_marking_for_individual_assignment', () => {
+it('Start Ustad Test Server ', () => {
+ // Start Test Server
+  cy.ustadStartTestServer()
+})
+
+it('Admin add a course and assignment block', () => {
+ // Admin user login
+  cy.ustadClearDbAndLogin('admin','testpass')
+ // Add a new course
+  cy.ustadAddCourse('004_012')
+ //Add a teacher
+  cy.contains("button","Members").click()
+  cy.contains("span","Add a teacher").click()
+  cy.ustadAddNewPerson('Teacher','1','Female')
+ // Add account for teacher
+  cy.contains("Teacher 1").click()
+  cy.contains('View profile').click()
+  cy.ustadCreateUserAccount('teacher1','test1234')
+ //Add a student1
+  cy.contains("span","Add a student").click()
+  cy.ustadAddNewPerson('Student','1','Male')
+  cy.contains("button","Members").should('be.visible')
+ //Add account for student1
+  cy.contains("Student 1").click()
+  cy.contains('View profile').click()
+  cy.ustadCreateUserAccount('student1','test1234')
+ //Add a student2
+  cy.contains("span","Add a student").click()
+  cy.ustadAddNewPerson('Student','2','Male')
+  cy.contains("button","Members").should('be.visible')
+ //Add account for student2
+  cy.contains("Student 2").click()
+  cy.contains('View profile').click()
+  cy.ustadCreateUserAccount('student2','test1234')
+})
+
+it('Teacher add assignment', () => {
+  cy.ustadClearDbAndLogin('teacher1','test1234')
+ // Add Assignment block
+  cy.contains("Course").click()
+  cy.contains("004_012").click()
+  cy.contains("button","Members").click() // This is a temporary command to make sure member list is loaded
+  cy.contains("button","Course").click()
+  cy.contains("button","Edit").click()
+  cy.contains("Add block").click()
+  cy.contains("Assignment").click()
+  cy.get('input[id="title"]').clear().type("Assignment 1")
+  cy.get('div[data-placeholder="Description"]').type("this is a simple assignment")
+  cy.contains("div","Graded").click()
+  cy.contains("li","Submitted").click()
+  cy.ustadSetDateTime(cy.get("#cbDeadlineDate"),  new Date(Date.now() + (1000*60*1000)))
+  cy.get('#caMarkingType').click()
+  cy.contains("li","Peers").click()
+  cy.get('#caPeerReviewerCount').should('exist')
+  cy.get('#caPeerReviewerCount').type('1')
+  cy.wait(1000) // wait to load the student list
```

Comment:
Please replace the wait command with cy.get using a timeout e.g. could look for the name of students